packages/store/test/test-encodeKey.js

```diff
@@ -114,3 +124,30 @@ test('order invariants', t => {
     }
   }
 });
+
+test('BigInt values round-trip', async t => {
+  await fc.assert(
+    fc.property(fc.bigInt(), n => {
+      const rt = decodeKey(encodeKey(n));
+      return assertionPassed(t.is(rt, n), () => rt === n);
+    }),
+  );
+});
+
+test('BigInt encoding comparison corresponds with numeric comparison', async t => {
+  await fc.assert(
+    fc.property(fc.bigInt(), fc.bigInt(), (a, b) => {
+      if (a === b) return assertionPassed(t.pass(), () => true);
+      const ea = encodeKey(a);
+      const eb = encodeKey(b);
+      if (a < b) {
+        return assertionPassed(t.true(ea < eb), () => ea < eb);
+      }
+      if (a > b) {
+        return assertionPassed(t.true(ea > eb), () => ea > eb);
+      }
+      t.fail(`Unexpected incomparable bigints: ${a} ${b}`);
+      return false;
```

Comment:
is this equivalent and more concise?

```suggestion
      const ea = encodeKey(a);
      const eb = encodeKey(b);
      t.is(a === b, ea === eb);
      t.is(a < b, ea < eb);
      t.is(a > b, ea > eb);
```

Reply:
Not quite, but I like the general approach. Adopted.